apps/server/src/models/users.ts

```diff
@@ -0,0 +1,43 @@
+import mongoose from 'mongoose';
+
+const usersSchema = new mongoose.Schema(
```

Comment:
Add the user type here
```suggestion
const usersSchema = new mongoose.Schema<User>(
```